### DISCUSSION TITLE
Should script tags work out of the box?

### DISCUSSION THREAD

**Discussion by hahnbeelee** [mdx-js/mdx#2317] [Q&A] [answered]
I'm trying to use [nbdev](https://nbdev.fast.ai/getting_started.html) to convert a jupyter notebook to mdx. nbdev converts to [quarto](https://quarto.org/) as an intermediary step which has a [docusaurus convertor](https://quarto.org/docs/output-formats/docusaurus.html) I've gotten the rest of the markdown to show properly but not the graphs. The graphs are huge chunks of javascript inside a script tag. Does mdx support script tags out of the box or do i have to do something special to get them to work?

**Comment by hahnbeelee** [comment]
My gut is telling me no because...

<img width="1133" alt="CleanShot 2023-06-27 at 21 58 33@2x" src="https://github.com/mdx-js/mdx/assets/55263191/b0587100-869e-41d5-a694-505a8dbe728a">

it looks like this docusaurus syntax is basically saying everything below 
<img width="192" alt="CleanShot 2023-06-27 at 21 59 48@2x" src="https://github.com/mdx-js/mdx/assets/55263191/400d1f2c-a7de-495a-ab7c-bbbe40f32ef8">
is html and this is something custom that docusaurus set up.

**Reply by hahnbeelee** [reply]
and because of this
https://github.com/withastro/astro/issues/6327

**Reply by wooorm** [reply, marked as answer]
We don‘t do anything special with scripts; whether scripts work has more to do with what framework you use (e.g., React). So that depends on them.
